help/deploy/complete-setup/configure-auth.md

```diff
@@ -3,75 +3,94 @@ title: "Configure authentication"
-3. Set 'Client Id' and 'Secret' if applicable. You can get it from your OpenID provider
-4. Make sure the correct Web Root is set in 'Admin' section
+1. On the left **Sidebar** go to **Settings** > **Users and Sessions**. Authentication settings page appears
+2. Enable *Facebook authentication* to use the Facebook Oauth method
+3. Set *Facebook Client Id* and *Facebook Secret* if applicable. You can get it from [Facebook](https://help.vtex.com/tutorial/adding-a-client-id-and-a-client-secret-to-log-in-with-facebook--3R7rzXWG1GswWOIkYyy8SO)
+4. Make sure the correct **Web Root** is set in the left **Sidebar** > **Settings** > **Admin** section
+
+### Github authentication
+
+1. On the left **Sidebar** go to **Settings** > **Users and Sessions**. Authentication settings page appears
+2. Enable *GitHub authentication* to use the Facebook Oauth method
+3. Set *GitHub Client Id* and *GitHub Secret* if applicable. You can get it from [GiHub](https://episyche.com/blog/how-to-create-oauth-client-id-and-client-secret-for-github)
+4. Make sure the correct **Web Root** is set in the left **Sidebar** > **Settings** > **Admin** section
 
 ## OpenID authentication
 
 Datagrok supports the OpenID protocol to allow users to be authenticated using OpenID providers, for example, Azure AD.
 
-1. Go to the Datagrok Settings section 'Users and Sessions'; this section contains all authentication settings.
-2. Enable 'Open Id authentication' to use the OpenID method
-3. Get a well-known-configuration route and set it to 'Open Id Config Endpoint'. It should look
+To setup OpenID authentication:
+
+1. On the left **Sidebar** go to **Settings** > **Users and Sessions**. Authentication settings page appears.
+2. Enable *Open Id authentication* to use the OpenID method
+3. Get a well-known-configuration route and set it to **Open Id Config Endpoint**. It should look
    like `https://login.datagrok.ai/.well-known/openid-configuration`
-4. Set 'Open Id Client Id' and 'Open Id Secret' as in your OpenId provider
-5. Set the' Open Id Code Challenge method' if you enabled authorization code encryption. In most cases, it is `S256`
-6. Set 'Open Id Login Claim', 'Open Id Email Claim', 'Open Id First Name Claim', and 'Open Id Last Name Claim' to
-   provide optional claims for the application
-7. OpenID auto-login can be enabled using the 'Open Id Auto Login' option
-8. Make sure the correct Web Root is set in 'Admin' section
+4. Set *Open Id Client Id* and *Open Id Secret* as in your OpenId provider
+5. Set the *Open Id Code Challenge method* if you enabled authorization code encryption. In most cases, it is `S256`
+6. Set *Open Id Login Claim*, *Open Id Email Claim*, *Open Id First Name Claim*, and *Open Id Last Name Claim* to provide optional claims for the application
+7. You can enable/disable OpenID auto-login using the *Open Id Auto Login* checkbox
+8. Make sure the correct **Web Root** is set in the left **Sidebar** > **Settings** > **Admin** section
```

Comment:
It is repeated in all instructions, lets move it to the top and make a general thing